### ISSUE TITLE
PostAsync_ReuseRequestContent_Success fails on OSX 10.13

### ISSUE THREAD

**Issue by davidsh** [dotnet/corefx#31104]
Found during PR #31100.

HttpClientHandlerTest.PostAsync_ReuseRequestContent_Success is failing on OSX 10.13 in CurlHandler:

https://mc.dot.net/#/user/davidsh/pr~2Fjenkins~2Fdotnet~2Fcorefx~2Fmaster~2F/test~2Ffunctional~2Fcli~2F/b14a215bc5e5263683e2ec248d2178785f98524a/workItem/System.Net.Http.Functional.Tests/analysis/xunit/System.Net.Http.Functional.Tests.PlatformHandler_HttpClientHandlerTest~2FPostAsync_ReuseRequestContent_Success(remoteServer:%20https:~2F~2Fcorefx-net-http2.azurewebsites.net~2FEcho.ashx)

```
Assert.Contains() Failure
Not found: This is the content string.
In value:  
```

**Comment by stephentoub** [comment]
Consistently or sporadically?

**Comment by davidsh** [comment]
>Consistently or sporadically?

During my PR it failed 3 times out of 3 times.

**Comment by wfurt** [comment]
This did not fail in last 3 month (Kusto history)
Closing for now.

**Comment by stephentoub** [comment]
It didn't fail because it was disabled:
https://github.com/dotnet/runtime/blob/26298bc6295be9cddfa185f9d1dc8e99a5c92278/src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.cs#L2218-L2221

**Comment by karelz** [comment]
Original bug was addressed in 3.0 - marking as such.
